tests/lib/rules/prefer-destructuring.js

```diff
@@ -109,6 +111,7 @@ ruleTester.run("prefer-destructuring", rule, {
         },
         {
             code: "var foo = object[bar];",
+            output: "var { bar: foo } = object;",
```

Comment:
This output is incorrect. It will access the property `"bar"`, whereas the initial code will access the property specified by the `bar` variable.

The correct output would be

```js
var { [bar]: foo } = object;
```